demos/BMW_demo/bmw_share.js

```diff
@@ -0,0 +1,164 @@
+
+function bmw_compute_share(jiff,input,n, threshold, Zp) {
```

Comment:
let's add documentation for this as well. You could copy/paste the standard jiff library 'compute_share' docstring and update it to reflect the name/inputs/result of this function.